gpt4all-chat/qml/ChatView.qml

```diff
@@ -478,86 +480,67 @@ Rectangle {
-
-                    onClicked: {
-                        Network.sendResetContext(chatModel.count)
-                        currentChat.reset();
-                        currentChat.processSystemPrompt();
-                    }
-                }
+                Layout.fillWidth: true
+                Layout.preferredWidth: 100
+                Layout.preferredHeight: childrenRect.height
+                Layout.topMargin: 20
 
-                MyToolButton {
-                    id: copyButton
-                    backgroundColor: theme.iconBackgroundLight
-                    width: 40
-                    height: 40
-                    z: 200
-                    padding: 15
-                    source: "qrc:/gpt4all/icons/copy.svg"
-                    Accessible.name: qsTr("Copy")
-                    Accessible.description: qsTr("Copy the conversation to the clipboard")
+                RowLayout {
+                    spacing: 20
+                    anchors.right: parent.right
+                    MyButton {
+                        id: collectionsButton
+                        Image {
+                            id: collectionsImage
+                            anchors.verticalCenter: parent.verticalCenter
+                            anchors.left: parent.left
+                            anchors.leftMargin: 15
+                            width: 24
+                            height: 24
+                            mipmap: true
+                            source: "qrc:/gpt4all/icons/db.svg"
+                        }
 
-                    TextEdit{
-                        id: copyEdit
-                        visible: false
-                    }
+                        ColorOverlay {
+                            anchors.fill: collectionsImage
+                            source: collectionsImage
+                            color: collectionsButton.hovered || collectionsImage.toggled ? theme.iconBackgroundHovered : theme.iconBackgroundLight
+                        }
 
-                    onClicked: {
-                        var conversation = getConversation()
-                        copyEdit.text = conversation
-                        copyEdit.selectAll()
-                        copyEdit.copy()
-                        copyMessage.open()
-                    }
-                }
+                        leftPadding: 50
+                        borderWidth: 0
+                        backgroundColor: theme.mainComboBackground
+                        backgroundColorHovered: theme.conversationButtonBackgroundHovered
+                        backgroundRadius: 5
+                        padding: 15
+                        topPadding: 8
+                        bottomPadding: 8
+                        textColor: hovered || toggled ? theme.iconBackgroundHovered : theme.iconBackgroundLight
+                        text: qsTr("local docs")
```

Comment:
We call this button "local docs", but in settings we call it "LocalDocs". And this seems to be the only button in the UI with a label that starts with a lowercase letter, aside from the links at the top.

Reply:
Hmm, yes I will change these to be consistent. Not sure but probably just 'Local docs' is best?